### ISSUE TITLE
Un-specific cutting 

### ISSUE THREAD

**Issue by lzy604** [vdemichev/DiaNN#679]
Dear DiaNN developer,

I used the command line to run the DIANN, and I want to know if the DIANN can be used to detect the un-specific cutting,  which means every kind of cleavage might exist.

--cut [specificty 1],[specificity 2],... specifies cleavage specificity for the in silico digest. Cleavage sites (pairs of amino acids) are listed separated by commas, '*' indicates any amino acid, and '!' indicates that the respective site will not be cleaved. Examples: "--cut K*,R*,!*P" - canonical tryptic specificity, "--cut " - digest disabled

I wonder the digest disabled mode is the meaning of a long protein do not digest any more and they are very long ones?

**Comment by LeeSCantrell** [comment]
@nuan604 , this is also available as a command line flag, --semi. You can add this in the additional options box in the windows GUI. Its listed as experimental and as Vadim mentions, search space may explode with this flag. MSFragger is a good option, but give both a shot in case computational availability isn't a problem. DIA-NN does very well with search space expansion.

**Comment by vdemichev** [comment]
No, non-specific digest searches will likely not work well with DIA-NN. I would suggest to try MSFragger-DIA which is part of FragPipe for this, but in general it's not the best idea for DIA, regardless of the processing software.
